core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-card/dot-pages-card.component.scss

```diff
@@ -0,0 +1,44 @@
+@use "variables" as *;
+
+::ng-deep {
+    .p-card .p-card-header {
+        padding: 0;
+    }
+
+    .p-card-content {
+        display: flex;
+    }
+
+    .dot-favorite-page-highlight ::ng-deep i {
+        color: $brand-primary;
+    }
+
+    .dot-pages-favorite-card-content__image {
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: contain;
+        padding-bottom: 75.25%;
```

Comment:
If this is the hack to keep the aspect ratio there is a native CSS now https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio

Reply:
thanks, but it can not be applied on this case